Lib/test/test_cmath.py

```diff
@@ -536,9 +534,6 @@ def test_abs(self):
         self.assertEqual(abs(complex(INF, NAN)), INF)
         self.assertTrue(math.isnan(abs(complex(NAN, NAN))))
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
-    @requires_IEEE_754
```

Comment:
The upper 2 lines are our temporary change for test, but the last line is not.
Please put `@requires_IEEE_754` back 😄

Reply:
Got it! I restored the `@requires_IEEE_754 ` decorator :)